android/src/main/java/com/airbnb/android/react/maps/AirMapModule.java

```
@@ -0,0 +1,126 @@
+                            return;
+                        }
+
+                        // Save the snapshot to disk
+                        OutputStream outputStream = null;
+                        try {
+                            if (result.equals(SNAPSHOT_RESULT_FILE)) {
+                                File tempFile = File.createTempFile("AirMapSnapshot", "." + format, context.getCacheDir());
+                                outputStream = new FileOutputStream(tempFile);
+                                snapshot.compress(compressFormat, (int)(100.0 * quality), outputStream);
+                                outputStream.close();
+                                outputStream = null;
+                                String uri = Uri.fromFile(tempFile).toString();
+                                promise.resolve(uri);
+                            }
+                            else if (result.equals(SNAPSHOT_RESULT_BASE64)) {
+                                outputStream = new ByteArrayOutputStream();
+                                snapshot.compress(compressFormat, (int)(100.0 * quality), outputStream);
+                                outputStream.close();
+                                outputStream = null;
+                                byte[] bytes = ((ByteArrayOutputStream) outputStream).toByteArray();
+                                String data = Base64.encodeToString(bytes, Base64.NO_WRAP);
+                                promise.resolve(data);
+                            }
+                        }
+                        catch (Exception e) {
+                            promise.reject(e);
+                        }
+                        finally {
+                            if (outputStream != null) {
+                                try {
+                                    outputStream.close();
+                                } catch (IOException e) {
+                                    e.printStackTrace();
```

remove this line, just ignore the exception